### ISSUE TITLE
Location of conftest.py

### ISSUE THREAD

**Issue by sofroniewn** [napari/napari#1314]
## 🚀 Feature
Definitely not the most important, but I find the current location of `conftest.py` on the top of the repo both hard to remember where it is and a little polluting of the top level. Could we move it to `_tests/conftest.py`.

I saw some things on stack overflow etc, and some docs here https://docs.pytest.org/en/2.7.3/plugins.html that imply we'd then need to import `confest.p` in the main `__init__.py` though which may be undesirable to be discovered everywhere, so if this isn't possible I'll live with it, but just wanted to see.

**Comment by tlambert03** [comment]
yeah, it's not ideal. I'm fine with any location as long as the tests work! :)

**Comment by willingc** [comment]
I'm doing some issue triage. I think we can go ahead and close this issue as not planned. Right now the tests are running well.
